### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by sa1nt9** [codewars/codewars.com#3102] [Q&A] [answered]
Hello. My account was blocked, why?

**Comment by Blind4Basics** [comment, marked as answer]
Hello,

(note: § help for readability ;) )

Rewriting some variables here and there (or so) doesn't make you author of an original solution. Yes, some of your next solutions were not obvious copies, but they still were copies.
Simplest course of action: delete your current account and start fresh. This time, don't try to copy/hide the copies, please.

Cheers
